src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -309,15 +309,15 @@ def _execute_command(kwargs):
             else:
                 return result
         except ClientException as client_exception:
-            log_telemetry('client exception', log_type='trace')
+            telemetry.set_exception(client_exception, 'client exception')
```

Comment:
The new API should be always exception first. If there are invocation other way around it's a bug.